mesop/dataclass_utils/dataclass_utils_test.py

```diff
@@ -49,6 +50,16 @@ class WithUploadedFile:
   data: UploadedFile = field(default_factory=UploadedFile)
 
 
+class PydanticModel(BaseModel):
```

Comment:
Can we add a more complicated example:

1. Nested PydanticModel
2. List of PydanticModels
3. More complicated data types / annotations in the PydanticModel

Reply:
good idea - done.